aten/src/ATen/native/nested/NestedTensorUtils.h

```diff
@@ -220,8 +219,8 @@ namespace impl {
 template <typename T>
 struct NestedNode {
   NestedNode() = delete;
-  explicit NestedNode(std::vector<T>&& children)
-      : _is_leaf(false), _children(children) {}
+  explicit NestedNode(std::vector<T> children)
```

Comment:
I mean, yu could still leave it as a universal reference if you wanted...

Reply:
For universal reference it still requires a move operation later..